### PR TITLE
WT-11135 measure and report code coverage per minute metrics

### PR DESCRIPTION
**Pull request by jeremythorp**

### CODE REVIEW

File: test/evergreen.yml [hunk 1]
```diff
@@ -3374,7 +3379,8 @@ tasks:
             source venv/bin/activate
             pip3 install lxml==4.8.0 Pygments==2.11.2 Jinja2==3.0.3 gcovr==5.0
             mkdir -p ../coverage_report
-            GCOV=/opt/mongodbtoolchain/v4/bin/gcov gcovr -r .. -f ../src -e '.*/bt_(debug|dump|misc|salvage|vrfy).*' -e '.*/(log|progress|verify_build|strerror|env_msg|err_file|cur_config|os_abort)\..*' -e '.*_stat\..*' -e 'bench' -e 'examples' -e 'test' -e 'ext' -e 'dist' -e 'tools' -j 4 --html-details --html-self-contained -o ../coverage_report/2_coverage_report.html
+            GCOV=/opt/mongodbtoolchain/v4/bin/gcov gcovr -r .. -f ../src -e '.*/bt_(debug|dump|misc|salvage|vrfy).*' -e '.*/(log|progress|verify_build|strerror|env_msg|err_file|cur_config|os_abort)\..*' -e '.*_stat\..*' -e 'bench' -e 'examples' -e 'test' -e 'ext' -e 'dist' -e 'tools' -j 4 --html-self-contained --html-details ../coverage_report/2_coverage_report.html --json-summary-pretty --json-summary ../coverage_report/1_coverage_report_summary.json
```
Comment (lukech) [inline test/evergreen.yml:3382]: I'm curious if a "-o" is needed in front of the HTML and json files generated by the gcovr command (it's used previously). Guess they are optional, just to confirm.

**Review by lukech — APPROVED** [review]
LGTM

File: test/evergreen.yml (hunk 1 above)
Comment (jeremythorp) [reply test/evergreen.yml:3382]: -o can be used to specify the output file. Alternatively the user provides the output file path immediately after the output format parameter (--html-details, and --json-summary in this case). By avoiding the -o option, and putting the output file path after each output format parameter, we can generate two different forms of output (into different files) in a single execution of gcovr.

Comment (lukech) [reply test/evergreen.yml:3382]: That's a really good tip to know, thanks!

**Pull request merged**